woocommerce/woocommerce-gutenberg-products-block
# Add transforms for shared attributes.

Fixes: #370 

This adds transforms for the 5 blocks which share attributes:

- woocommerce/product-best-sellers
- woocommerce/product-category
- woocommerce/product-new
- woocommerce/product-on-sale
- woocommerce/product-top-rated

#### Accessibility

No changes to any interfaces, just using existing Gutenberg transforms.

### How to test the changes in this Pull Request:

1. Create a block of one of the types, set columns, set categories
2. Transform it to each of the other types in turn. Each of the 20 transforms possible should be tested from each block to each other block.
